data/mods/Xedra_Evolved/eocs/inventor.json

```diff
@@ -58,6 +58,7 @@
           "binary_sword",
           "bio_launcher",
           "bio_warhead",
+          "portable_dark_age",
```

Comment:
There is an item group for inventor devices somewhere, such that occasionally samples can be found in Xedra evidence rooms, perhaps it should be there as well. On the other hand, maybe putting an EMP bomb of uncertain durability in a building full of delicate electronics sounds ... unadvisable, so I think it's better not to.

Reply:
I need to move the evidence rooms to the second floor so I'm hoping to add missing items at the same time as I move it.